syntax = "proto3";

import "adlinktech/datariver/descriptor.proto";

package com.adlinktech.example.protobuf;


message CameraState {
  option(.adlinktech.datariver.tag_group) = {
    qosProfile: "state"
    description: "ADLINK Edge SDK Example Camera State TagGroup"
  };

  string state = 1 [(.adlinktech.datariver.field_options) = {
    description: "Camera state"
    unit: "n/a"
  }];
};
